<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormNewProductContent</class>
 <widget class="QWidget" name="FormNewProductContent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="styleSheet">
      <string notr="true">QSplitter::handle {
    image: url(&quot;:/images/splitter_icon.png&quot;);
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>35</number>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="ButtonBeR" name="btn_zoom_in">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/zoom_in.png</normaloff>:/icons/zoom_in.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
           <property name="autoRepeatDelay">
            <number>600</number>
           </property>
           <property name="autoRepeatInterval">
            <number>250</number>
           </property>
           <property name="bgIcon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/bt-blank.png</normaloff>
             <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
           </property>
           <property name="bgIconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ButtonBeR" name="btn_zoom_out">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/zoom_out.png</normaloff>:/icons/zoom_out.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
           <property name="autoRepeatDelay">
            <number>600</number>
           </property>
           <property name="autoRepeatInterval">
            <number>250</number>
           </property>
           <property name="bgIcon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/bt-blank.png</normaloff>
             <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
           </property>
           <property name="bgIconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ButtonBeR" name="btn_zoom_fit">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/zoom_fit.png</normaloff>:/icons/zoom_fit.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
           <property name="autoRepeatDelay">
            <number>600</number>
           </property>
           <property name="autoRepeatInterval">
            <number>250</number>
           </property>
           <property name="bgIcon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/bt-blank.png</normaloff>
             <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
           </property>
           <property name="bgIconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ButtonBeR" name="btn_zoom_area">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/zoom_fit_all.png</normaloff>:/icons/zoom_fit_all.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
           <property name="autoRepeatDelay">
            <number>600</number>
           </property>
           <property name="autoRepeatInterval">
            <number>250</number>
           </property>
           <property name="bgIcon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/bt-blank.png</normaloff>
             <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
           </property>
           <property name="bgIconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="ButtonBeR" name="btn_SincronizeDataBase">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="icon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/database-reload-alt-2.png</normaloff>:/icons/database-reload-alt-2.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="bgIcon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/bt-blank.png</normaloff>
             <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
           </property>
           <property name="bgIconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="ButtonBeR" name="btn_camera_snap">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/camera.png</normaloff>:/icons/camera.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoRepeatDelay">
            <number>600</number>
           </property>
           <property name="autoRepeatInterval">
            <number>250</number>
           </property>
           <property name="bgIcon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/bt-blank.png</normaloff>
             <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
           </property>
           <property name="bgIconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>32</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="ImageDevice" name="imagedevice"/>
       </item>
      </layout>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>771</width>
         <height>68</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4"/>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Inspeções:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lst_tools">
       <property name="minimumSize">
        <size>
         <width>285</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>285</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>191</red>
             <green>197</green>
             <blue>200</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>191</red>
             <green>197</green>
             <blue>200</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>145</red>
             <green>145</green>
             <blue>145</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="ButtonBeR" name="btn_add">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res/icons.qrc">
           <normaloff>:/icons/zoom_in.png</normaloff>:/icons/zoom_in.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
         <property name="autoRepeatDelay">
          <number>600</number>
         </property>
         <property name="autoRepeatInterval">
          <number>250</number>
         </property>
         <property name="bgIcon">
          <iconset resource="../res/icons.qrc">
           <normaloff>:/icons/bt-blank.png</normaloff>
           <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
         </property>
         <property name="bgIconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ButtonBeR" name="btn_remove">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res/icons.qrc">
           <normaloff>:/icons/zoom_out.png</normaloff>:/icons/zoom_out.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
         <property name="autoRepeatDelay">
          <number>600</number>
         </property>
         <property name="autoRepeatInterval">
          <number>250</number>
         </property>
         <property name="bgIcon">
          <iconset resource="../res/icons.qrc">
           <normaloff>:/icons/bt-blank.png</normaloff>
           <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
         </property>
         <property name="bgIconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ButtonBeR" name="btn_up">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res/icons.qrc">
           <normaloff>:/icons/arrow-3-up_0.png</normaloff>:/icons/arrow-3-up_0.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
         <property name="autoRepeatDelay">
          <number>600</number>
         </property>
         <property name="autoRepeatInterval">
          <number>250</number>
         </property>
         <property name="bgIcon">
          <iconset resource="../res/icons.qrc">
           <normaloff>:/icons/bt-blank.png</normaloff>
           <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
         </property>
         <property name="bgIconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ButtonBeR" name="btn_down">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res/icons.qrc">
           <normaloff>:/icons/arrow-3-down_0.png</normaloff>:/icons/arrow-3-down_0.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
         <property name="autoRepeatDelay">
          <number>600</number>
         </property>
         <property name="autoRepeatInterval">
          <number>250</number>
         </property>
         <property name="bgIcon">
          <iconset resource="../res/icons.qrc">
           <normaloff>:/icons/bt-blank.png</normaloff>
           <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
         </property>
         <property name="bgIconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonBeR</class>
   <extends>QPushButton</extends>
   <header>buttonber.h</header>
  </customwidget>
  <customwidget>
   <class>ImageDevice</class>
   <extends>QGraphicsView</extends>
   <header>mv/imagedevice.h</header>
   <slots>
    <slot>zoomIn()</slot>
    <slot>zoomOut()</slot>
    <slot>zoomFit()</slot>
    <slot>zoomFitItems()</slot>
    <slot>lockParent(bool)</slot>
    <slot>snap()</slot>
    <slot>Show()</slot>
    <slot>Hide()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_zoom_in</sender>
   <signal>clicked()</signal>
   <receiver>imagedevice</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_zoom_out</sender>
   <signal>clicked()</signal>
   <receiver>imagedevice</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_zoom_fit</sender>
   <signal>clicked()</signal>
   <receiver>imagedevice</receiver>
   <slot>zoomFit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_zoom_area</sender>
   <signal>clicked()</signal>
   <receiver>imagedevice</receiver>
   <slot>zoomFitItems()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
